src/widgets/helper/NotebookTab.cpp

```diff
@@ -64,6 +64,13 @@ NotebookTab::NotebookTab(Notebook *notebook)
         this->notebook_->removePage(this->page);
     });
 
+    this->menu_.addAction("Popup", [=]() {
+        if (auto container = dynamic_cast<SplitContainer *>(this->page))
+        {
+            container->popup();
+        }
+    });
+
```

Comment:
Nitpicky addition of " Tab" at the end plus the QKeySequence from [this `QMenu::addAction` overload](https://doc.qt.io/qt-5/qmenu.html#addAction-2)
```suggestion
    this->menu_.addAction(
        "Popup Tab",
        [=]() {
            if (auto container = dynamic_cast<SplitContainer *>(this->page))
            {
                container->popup();
            }
        },
        QKeySequence("Ctrl+Shift+N"));
```

Reply:
Resolved in 01ccd7ae